skyportal/models.py

```diff
@@ -585,6 +585,14 @@ class Obj(Base, ha.Point):
         doc="Notifications regarding the object sent out by users",
     )
 
+    listings = relationship(
+        "Listing",
+        back_populates="obj",
+        cascade="delete",
+        passive_deletes=True,
+        doc="Listings like favorites list",
```

Comment:
```suggestion
        doc="Listings like favorites list",
```
more descriptive doc?